src/main/java/org/mifos/connector/gsmastub/api/BatchtransactionsApiController.java

```diff
@@ -1,8 +1,8 @@
 package org.mifos.connector.gsmastub.api;
 
 import com.fasterxml.jackson.databind.ObjectMapper;
+import io.swagger.v3.oas.annotations.*;
```

Comment:
can we use specific class imports here instead of wildcard?

Reply:
Just here more than 7 imports are there so better to use wildcard in this particular case